market.h

```diff
@@ -56,22 +61,8 @@ class Market : public AListElem {
 	int baseprice;
 	int activity;
 
-	void PostTurn(int, int);
-	void Writeout(ostream& f);
-	void Readin(istream& f);
-	//AString Report();
-};
-
-class MarketList : std::vector<Market*> {
```

Comment:
This wasn't used, *AND* deriving from STL containers is a bad idea since they don't have virtual destructors.